bin/i18n-codeorg/in.sh

```diff
@@ -28,19 +28,6 @@ cp_in $orig_dir/slides.en.yml $loc_dir/slides.yml
 cp_in $orig_dir/unplugged.en.yml $loc_dir/unplugged.yml
 cp_in $orig_dir/courses.en.yml $loc_dir/courses.yml
 
-### Apps
-
-orig_dir=apps/i18n
-loc_dir=i18n/locales/source/blockly-mooc
-mkdir -p $loc_dir
-
-# Copy JSON files.
-for file in $(find $orig_dir -name 'en_us.json'); do
-  relname=${file#$orig_dir}
-  cp_in $file $loc_dir${relname%/en_us.json}.json
-done
```

Comment:
Moved to: https://github.com/code-dot-org/code-dot-org/pull/52987/files#diff-5ae61b20780c655f17850699e160e686d6f532a81371d8be5c2057096ffe9238R15-R19